README.md

```diff
@@ -1,2 +1,102 @@
 # cli
+
 A command line client for interacting a secure mailbox
+
+## Requirements
+
+- [click](https://click.palletsprojects.com/en/7.x/)
+- [python-gnupg](https://pythonhosted.org/python-gnupg/)
+- [pipenv](https://pipenv.pypa.io/en/latest/)
+- [requests](https://requests.readthedocs.io/en/master/)
+
+## Getting Started
+
+### API
+
+Before use this application, install and run [api](https://github.com/securemailbox/api).
+
+### CLI
+
+```bash
+# To download and install
+pyenv install 3.8.1
+
+# To use
+pyenv global 3.8.1
+
+# To check
+python --version # Python 3.8.1
+
+# To install other requirements
+pipenv install
+```
+
+## Features
+
+### Create key
+
+```bash
+pipenv run python scmail.py create-key
+```
+
+options:
+
+- name
+- email
+- key_type
+- key_length
+- expire_date
+- password
+
+### Register
+
+```bash
+pipenv run python sc_mail.py register
```

Comment:
```suggestion
pipenv run python scmail.py register
```

Reply:
Thanks, same type error...
So I just reply you one times.